src/BloomExe/Publish/MakePdfUsingGeckofxHtmlToPdfProgram.cs

```diff
@@ -4,6 +4,7 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Printing;
```

Comment:
I think this needs #if !__MonoCS__  /  #endif  around it.  Mono doesn't have a System.Printing.dll assembly, so it may not have that namespace defined.